src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,21 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+public class ListCommand extends Command {
+    /**
+     * Executes a list command using the given task list, UI and file storage.
+     *
+     * @param tasks the task list supplied.
+     * @param ui the UI supplied.
+     * @param storage the file storage supplied.
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        StringBuilder myBuilder = new StringBuilder();
+        myBuilder.append("Here are the tasks in your list:\n");
+        myBuilder.append(tasks.toString());
```

Comment:
myBuilder in line 18 could be omitted as it has already been specified in line 17
```suggestion
        myBuilder.append("Here are the tasks in your list:\n")
                         .append(tasks.toString());
```

Reply:
Thank you for the advice! Never noticed that `append` returns a reference to the `StringBuilder` object itself before.